Cookbook/Technical-Documents/BackendFeatureHowTo.md

```diff
@@ -0,0 +1,200 @@
+# How to implement a feature that works with the backend
+Let's look at a minimal and simple example of working with the backend to get some items into our view model, before moving on to more difficult topics.
+
+## A simple example
+Imagine some endpoint on the backend returns this piece of JSON:
+```
```

Comment:
nit: you should use `json` after the 3 backticks to have JSON syntax highlighting of the snippet

Reply:
Ah right, I keep forgetting about it :)